app/src/main/java/com/ddoczi/tasky/agenda/presentation/detail/composables/DetailNotificationSelector.kt

```diff
@@ -0,0 +1,90 @@
+package com.ddoczi.tasky.agenda.presentation.detail.composables
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.NavigateNext
+import androidx.compose.material.icons.filled.Notifications
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.DarkGray
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.ddoczi.tasky.R
+import com.ddoczi.tasky.core.presentation.composables.TaskyDropdown
+import com.ddoczi.tasky.ui.theme.Black
+import com.ddoczi.tasky.ui.theme.Gray
+
+@Composable
+fun DetailNotificationReminder(
```

Comment:
Ah, I see it's about notification **reminders** 😄 I'd reflect that in the agenda events as well by calling it something like `OnNotificationReminderClick` or just something like `OnReminderTypeClick`